hazelcast/src/main/java/com/hazelcast/internal/networking/nio/NioNetworking.java

```diff
@@ -127,6 +128,8 @@ public NioNetworking(Context ctx) {
         this.selectorWorkaroundTest = ctx.selectorWorkaroundTest;
         this.idleStrategy = ctx.idleStrategy;
         metricsRegistry.scanAndRegister(this, "tcp");
+        outboundNetworkStats.registerMetrics(metricsRegistry, "tcp.bytesSend");
```

Comment:
Potential misprint. Shouldn't it be `tcp.bytesSent`?

Reply:
In other places it's used as `bytesSend`. Didn't want to deviate from the conventions used here and change other places in a patch release. I can fix this in the forward port (both this and other such usages).